.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug/Performance report
+about: Create a report to help us improve
+title: "[Bug/Performance]"
+labels:
+assignees: ''
+
+---
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+**System information**
```

Comment:
I prefer to let user fill `system information` at first ,and then `describe the bug`.

Reply:
okay system info is moved up